src/arrays.js

```diff
@@ -1,61 +1,73 @@
 const getNthElement = (index, array) => {
-  // your code here
+  
+  if (index >= array.length){
+    return array[index-array.length];
+  }
+  return array[index];
 };
```

Comment:
although this works for the tests provided, this will give an incorrect result if the index greater than 2 times the length of the array. A better solution would be to use the modulus `%` operator. i.e.
```
const getNthElement = (index, array) => {
  return array[index % array.length];
};
```
Try to have a think about why this works, and maybe play about with it to see if you can figure out the logic